assets/react/v3/entries/course-builder/components/modals/AddLessonModal.tsx

```diff
@@ -98,24 +109,24 @@ const AddLessonModal = ({ closeModal, icon, title, subtitle }: AddLessonModalPro
               />
             )}
           />
-          {/* // @TODO: Need to add FormVideo component when its implemented*/}
-          {/* <Controller
-              name='thumbnail'
-              control={form.control}
-              render={(controllerProps) => (
-                <FormImageInput
-                  {...controllerProps}
-                  label={__('Featured Image', 'tutor')}
-                  buttonText={__('Upload Course Thumbnail', 'tutor')}
-                  infoText={__('Size: 700x430 pixels', 'tutor')}
-                />
-              )}
-            /> */}
+          <Controller
+            name="video"
+            control={form.control}
+            render={(controllerProps) => (
+              <FormVideoInput
+                {...controllerProps}
+                label={__('Video', 'tutor')}
+                buttonText={__('Upload Video', 'tutor')}
+                infoText={__('Supported file formats .mp4', 'tutor')}
```

Comment:
Only mp4??

Reply:
That's how it is given in design and current TUTOR LMS